scripts/linters/general_purpose_linter.py

```diff
@@ -293,14 +284,27 @@
     {
         'regexp': re.compile(r'\$parent'),
         'message': 'Please do not access parent properties ' +
-                   'using $parent. Use the scope object' +
+                   'using $parent. Use the scope object ' +
                    'for this purpose.',
         'excluded_files': (),
         'excluded_dirs': ()
     }
 ]
 
 BAD_PATTERNS_PYTHON_REGEXP = [
+    {
+        'regexp': re.compile(r'__author__'),
+        'message': 'Please remove author tags from this file.',
+        'excluded_files': (),
```

Comment:
Why these changes?

Reply:
__author__ tag is only used in python only, so I moved it to python regexes

Reply:
Why regex, earlier it was a simple string check right? Which is comparatively faster, right?

Reply:
Yeah but before it is checking all the files including HTML, CSS, TS etc, which is taking extra time, now it will only check python file.

Reply:
I see, that's a goof inding, thanks for moving it.